### DISCUSSION TITLE
@antv/s2 透视表如何开启组内排序

### DISCUSSION THREAD

**Discussion by Kitesource** [antvis/S2#2321] [Q&A]
help, 开启了配置 showDefaultHeaderActionIcon: true， 组内排序的弹框没有显示出来

**Comment by lijinke666** [comment]
https://s2.antv.antgroup.com/manual/faq#%E4%B8%BA%E4%BB%80%E4%B9%88-tooltip-%E5%9C%A8-antvs2-%E4%B8%AD%E4%B8%8D%E6%98%BE%E7%A4%BA%E5%9C%A8-antvs2-react-antvs2-vue-%E4%B8%AD%E5%8F%AF%E4%BB%A5%E6%AD%A3%E5%B8%B8%E6%98%BE%E7%A4%BA

**Reply by Kitesource** [reply]
所以是要自定义tooltip， 自己实现排序对吗？

**Reply by lijinke666** [reply]
是的, @antv/s2 是 core 层, 提供表格渲染和排序的能力, @antv/s2-react 基于它封装了一些分析组件, 比如 排序菜单/tooltip, 所以如果你不依赖框架的话, 就需要自定义 tooltip, 调用 s2 提供的排序方法

可以参考 https://github.com/antvis/S2/discussions/1280
